pica2mods-xslt/src/main/resources/xsl/default/pica2mods-default-subject.xsl

```diff
@@ -26,15 +26,15 @@
 
   <xsl:template name="modsSubject">
     <xsl:for-each select="./p:datafield[@tag='045R' and p:subfield[@code = 9]]/p:subfield[@code = 'k']">
-      <mods:subject>
+      <mods:subject authority="k10plus_field_045R">
```

Comment:
Können wir hier, wie bei den anderen Schlagworten auch, die Pica3-Notation verwenden ("5090")
siehe https://format.k10plus.de/k10plushelp.pl?cmd=ppindex&katalog=Standard

sonst müssten wir "reden" und das einheitlich umstellen.

Reply:
ich habe bisher nur pica+ gesprochen aber ich bin in der Sache emotionslos